Equals.Fody/Equals.Fody.csproj

```
@@ -4,7 +4,13 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <DebugType>portable</DebugType>
   </PropertyGroup>
+  <PropertyGroup>
+    <PackageTags>Equatable;Equals;Fody;Weave;CodeGen</PackageTags>
+    <Description>
+      Generate Equals, GetHashCode and operators methods from properties for class decorated with a [Equals] Attribute
```

```suggestion
      Generate Equals, GetHashCode and operator methods from properties for classes decorated with an [Equals] Attribute
```

I'm not a native English speaker, but I think that's more correct (and yeah. the same sentence is in the readme...)